package.json

```diff
@@ -24,22 +24,22 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "bootstrap": "^4.1.3",
+    "bootstrap": "^4.3",
```

Comment:
can we please fix the bootstrap version? we don't want to have that upgraded without our knowledge.

Reply:
and I guess that goes the same for all our dependencies

Reply:
Not sure what you mean. The purpose of this PR is to change the version to 4.3, so that's what was changed here.

Reply:
Ah, I didn't get what you meant by "fix". I guess we should change to at least "~4.3", but I don't think using exactly "4.3" is necessary, do you?

Reply:
I would do it just to be on the safe side. breaking changes happen all the time

Reply:
maybe we can address this too in another PR